rc/servers.kak

```diff
@@ -511,6 +511,21 @@ hook -group lsp-filetype-toml global BufSetOption filetype=toml %{
     }
 }
 
+hook -group lsp-filetype-typst global BufSetOption filetype=typst %{
+    set-option buffer lsp_servers %exp{
```

Comment:
thanks! I think we should use `""` for now instead of `%exp{}` because we still support Kakoune v2021.

Reply:
sure! would i just replace the %exp{ } with " ", or should i escape what's inside also?

Reply:
One way is to replace, and double up any `"` inside (see also `;doc expansions`). It's very ugly unfortunately.
So how about a different approach:

```
set-option buffer lsp_servers %{ ... }
set-option -add buffer lsp_servers "formatterPrintWidth = %opt{autowrap_column}"
```

if that's not too confusing..